test/JDBC/expected/babel_time-vu-verify.out

```diff
@@ -0,0 +1,925 @@
+-- psql
```

Comment:
let us add cases where we have invalid date/time values (too small, too large). Also get date/time from a table column rather than as expression. Add cases where expression comes after applying some operator like case, +/- etc.

Reply:
Sure, will add that in next revision.